SpaceCore/Patches/HoeDirtPatcher.cs

```diff
@@ -13,7 +13,6 @@
 namespace SpaceCore.Patches
 {
     /// <summary>Applies Harmony patches to <see cref="HoeDirt"/>.</summary>
-    [SuppressMessage("ReSharper", "InconsistentNaming", Justification = "The naming is determined by Harmony.")]
```

Comment:
My VS2019 flagged this as not needed.

Reply:
It suppresses a warning from [ReSharper](https://www.jetbrains.com/resharper/), so it doesn't apply if you don't have that installed. (I'll just restore that line before merging.)